css/css-conditional/at-supports-selector-webkit-autofill.optional.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<title>CSS Test (Conditional Rules): In @supports, known -webkit- pseudo elements can be parsed successfully</title>
+<link rel="match" href="at-supports-001-ref.html">
+<style>
+  div {
+    background:red;
+    height:100px;
+    width:100px;
+  }
+  @supports selector(input::-webkit-autofill) {
```

Comment:
Wait, I just noticed this isn't actually a known `-webkit-` pseudo-element. `:-webkit-autofill` is a pseudo-class, not a pseudo-element.

If you want to test an unknown `::-webkit-` pseudo-element, it should be `::-webkit-unknown-pseudo-element` or something.

Reply:
I want to test a known one. Hmm.

Reply:
I guess all pseudo-elements are proprietary at the moment so this only matters for browsers supporting a -webkit-prefixed pseudo-element. I guess I could pick one of the ones Chromium and WebKit both have.